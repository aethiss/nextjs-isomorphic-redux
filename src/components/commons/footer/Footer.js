import React from 'react'

// Styles
import styles from '../../../../styles/Home.module.css'

const Footer = () => {
  return (
    <footer className={styles.footer}>
      <a
        href='https://github.com/aethiss?tab=repositories'
        target='_blank'
        rel='noopener noreferrer'
      >
        Powered by aethiss@gmail.com (Git)
      </a>
    </footer>
  )
}

export default Footer
