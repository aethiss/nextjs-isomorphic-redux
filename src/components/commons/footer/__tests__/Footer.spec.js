import React from 'react'
import Footer from '../Footer'
import TestRenderer from 'react-test-renderer'

describe('Footer Component', () => {
  const testRenderer = TestRenderer.create(<Footer />)
  describe('Render', () => {
    it('[Footer] Default render', () => {
      expect(testRenderer).toMatchSnapshot()
    })
  })
})
