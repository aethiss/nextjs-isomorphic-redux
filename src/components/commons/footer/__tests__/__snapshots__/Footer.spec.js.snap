// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Render [Footer] Default render 1`] = `
<footer
  className="footer"
>
  <a
    href="https://github.com/aethiss?tab=repositories"
    rel="noopener noreferrer"
    target="_blank"
  >
    Powered by aethiss@gmail.com (Git)
  </a>
</footer>
`;
